### ISSUE TITLE
'Wait for element present' missing value

### ISSUE THREAD

**Issue by mattmundell** [SeleniumHQ/selenium-ide#1276]
## 🐛 Bug Report

When I add a 'Wait for element present' command from the browser menu then the command value is blank.  This catches me quite often, because the test requires a timeout value.
Note that if I add the same command via the Selenium window command field, a timeout of 30000 is added.

## To Reproduce

1 open a project
2 start recording
3 in the browser window, right click on an element
4 select Selenium IDE > Wait For > Present

## Expected behavior

They created command has a timeout of 30000.

## Environment

OS: Ubuntu 20.04
Selenium IDE Version:  3.17.0
Selenium SIDE Runner Version:  
Node version: 
Browser: Chromium
Browser Version: 93.0.4577.82 (Official Build) snap (64-bit)

**Comment by toddtarsi** [comment]
Interesting. So, field defaults aren't populated from context menu step selections? This might be tricky.

**Comment by mattmundell** [comment]
Yes sounds right.  At least the other "wait for" commands behave the same.

Related: "wait for text" has a default of 30000 which doesn't make much sense (because it's the text being waited for).

**Comment by toddtarsi** [comment]
That is a good call out. I'll try taking a look here around early-mid December. For the next weekend or two, I'm just cleaning up some break-fix cases in the side-utils output.

**Comment by toddtarsi** [comment]
This defaults well on v4 latest

**Comment by toddtarsi** [comment]
Eh, maybe not
